packages/plugins/posthog/src/lib/posthog-core.module.ts

```
@@ -0,0 +1,87 @@
+
+		// Register useClass as a provider if defined
+		if (options.useClass) {
+			providers.push({
+				provide: options.useClass,
+				useClass: options.useClass
+			});
+		}
+
+		return providers;
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)